app/vmalert/config/parser.go

```diff
@@ -0,0 +1,12 @@
+package config
```

Comment:
why `config/parser.go` and not just `config/config.go`?

Reply:
It is just a parser submodule, we may need to use config.go for reload of rules or configuration or any other logic.